package.json

```diff
@@ -12,7 +12,8 @@
     "server": "nodemon server/server.js",
     "test": "jest --watch *.js",
     "enzymetest": "react-scripts test --env=jsdom",
-    "seed": "node server/seed.js"
+    "seed": "node server/seed.js",
+    "docker": "concurrently \"npm run seed\" \"npm run server\""
```

Comment:
Interesting, I thought I'd have to run the seed script during the build phase and not for execute. That's what I was doing incorrectly in the beginning.